aws-java-sdk-core/src/main/java/com/amazonaws/retry/PredefinedRetryPolicies.java

```diff
@@ -199,6 +198,10 @@ public boolean shouldRetry(AmazonWebServiceRequest originalRequest,
         /** Maximum exponential back-off time before retrying a request */
         private static final int MAX_BACKOFF_IN_MILLISECONDS = 20 * 1000;
 
+        /** Maximum number of retries before the max backoff will be hit */
+        private static final int MAX_RETRIES_BEFORE_MAX_BACKOFF =
```

Comment:
I'd vote for simplicity and change the constant when the other values are changed, instead of having readability to suffer.